packages/lib/delivery/src/notifications/channels/Push.ts

```diff
@@ -3,21 +3,40 @@ import webpush from 'web-push';
 import { NotificationType } from '../types';
 
 type UserWebPushConfig = {
-    recipientValue: webpush.PushSubscription;
+    recipientValue: string;
     notificationType: NotificationType;
     notificationContent: any; // object that contains notification details
 };
 
+type PushNotificationServerConfig = {
+    vapidEmailId: string;
+    publicVapidKey: string;
+    privateVapidKey: string;
+};
+
 // method to send web push notification
-export function Push() {
+export function Push(serverConfig: PushNotificationServerConfig) {
     const send = async (config: UserWebPushConfig) => {
         try {
+            const options = {
+                vapidDetails: {
+                    subject: serverConfig.vapidEmailId,
+                    publicKey: serverConfig.publicVapidKey,
+                    privateKey: serverConfig.privateVapidKey,
+                },
+            };
+
             const payload = JSON.stringify({
                 body: `You received a new message from ${config.notificationContent.from}.`,
                 title: 'New Message',
+                image: 'https://framerusercontent.com/images/6kPsfvAEqACFXgMOWU539mUqfB0.png',
```

Comment:
The browser notification that user will see like this 
![Screenshot from 2024-06-10 14-44-11](https://github.com/dm3-org/dm3/assets/137804674/a15b3c2c-afc9-4121-a7c6-e7ecb853123a)
There is a logo of DM3, this URL is for that logo only. I took this URL from our website https://dm3.network/
This will be changed as we will decide in future.